mappings/net/minecraft/world/gen/noise/NoiseRouter.mapping

```diff
@@ -4,28 +4,54 @@ CLASS net/minecraft/unmapped/C_fpiyqmft net/minecraft/world/gen/noise/NoiseRoute
 	FIELD f_dnucagsv depth Lnet/minecraft/unmapped/C_kzwapyud;
 	FIELD f_fgotwyuw erosion Lnet/minecraft/unmapped/C_kzwapyud;
 	FIELD f_fyvhcwjp fluidLevelFloodNoise Lnet/minecraft/unmapped/C_kzwapyud;
-	FIELD f_hcldmwhc weirdness Lnet/minecraft/unmapped/C_kzwapyud;
-	FIELD f_hqbjcvhs continentalness Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_hcldmwhc ridges Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_hqbjcvhs continents Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
These are the official terms used for e.g. datapacks, personally I wouldn't want these changed

Reply:
The codec fields were renamed to these